### PR TITLE
Ria 3130 final bundling to pre hearing

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-3130

### Change description ###

Final bundling to prehearing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Comment by makowalski** [comment]
Sonar check fails on Jenkins build, please fix

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/CustomiseHearingBundlePreparer.java [hunk 1]
```diff
@@ -47,26 +49,54 @@ public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCa
...
+            prepareCustomDocuments(asylumCase, FTPA_APPELLANT_EVIDENCE_DOCUMENTS, CUSTOM_FTPA_APPELLANT_EVIDENCE_DOCS);
+            prepareCustomDocuments(asylumCase, FTPA_RESPONDENT_EVIDENCE_DOCUMENTS, CUSTOM_FTPA_RESPONDENT_EVIDENCE_DOCS);
+            prepareCustomDocuments(asylumCase, FTPA_APPELLANT_DOCUMENTS, CUSTOM_FTPA_APPELLANT_DOCS);
+            prepareCustomDocuments(asylumCase, FTPA_RESPONDENT_DOCUMENTS, CUSTOM_FTPA_RESPONDENT_DOCS);
+            prepareCustomDocuments(asylumCase, FINAL_DECISION_AND_REASONS_DOCUMENTS, CUSTOM_FINAL_DECISION_AND_REASONS_DOCS);
+            prepareCustomDocuments(asylumCase, REHEARD_HEARING_DOCUMENTS, CUSTOM_REHEARD_HEARING_DOCS);
+            prepareCustomDocuments(asylumCase, ADDENDUM_EVIDENCE_DOCUMENTS, CUSTOM_APP_ADDENDUM_EVIDENCE_DOCS);
+            prepareCustomDocuments(asylumCase, ADDENDUM_EVIDENCE_DOCUMENTS, CUSTOM_RESP_ADDENDUM_EVIDENCE_DOCS);
+
+            //if (asylumCase.read(ADDENDUM_EVIDENCE_DOCUMENTS).isPresent()) {
```
Comment (makowalski) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/CustomiseHearingBundlePreparer.java:67]: this can be removed

**Review by makowalski — DISMISSED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/CustomiseHearingBundlePreparer.java (hunk 1 above)
Comment (mkayad) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/CustomiseHearingBundlePreparer.java:67]: these collections are used for the customisation of the hearing.

**Review by makowalski — DISMISSED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**